src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/PagingQueryString.feature.disabled

```diff
@@ -0,0 +1,72 @@
+                  | 6         | UT Austin College of Education Under Graduate | [ "Postsecondary" ] | [ "Educator Preparation Provider" ] |
+                  | 7         | UT Austin Extended Campus                     | [ "Postsecondary" ] | [ "Educator Preparation Provider" ] |
+                  | 255901001 | Grand Bend High School                        | [ "Tenth grade" ]   | [ "School" ]                        |
+                  | 255901044 | Grand Bend Middle School                      | [ "Seventh grade" ] | [ "School" ]                        |
+
+        @ignore
+        Scenario: Ensure clients can't GET information when filtering using invalid limit
+             When sending a GET request to "/ed-fi/schools?limit=<Value>"
+             Then it should respond with 400
+              And response body is
+                  """
+                  {"detail": "The limit parameter was incorrect.","type": "urn:ed-fi:api:bad-request:parameter","title": "Parameter Validation Failed","status": 400,"correlationId": null,"errors": ["Limit must be omitted or set to a value between 0 and 500."]}
+                  """
+        Examples:
+                  | Value                    |
+                  | -1                       |
+                  | 'zero'                   |
+                  | '5; select * from users' |
+                  | '0)'                     |
+                  | '1%27'                   |
+
+        @ignore
+        Scenario: Ensure clients can get information when filtering by limit and and a valid offset
+             When sending a GET request to "/ed-fi/schools?offset=3&limit=5"
+             Then it should respond with status 200
+              And the response array contains 2 items
+
+        @ignore
+        Scenario: Ensure clients can get information when filtering by limit and offset greater than the total
+             When sending a GET request to "/ed-fi/schools?offset=6&limit=5"
+             Then it should respond with status 200
+              And response array contains 0 items
+
+        @ignore
+        Scenario: Ensure clients can't GET information when filtering by limit and offset using invalid offset
+             When sending a GET request to "/ed-fi/schools?limit=6&offset=<Value>"
+             Then it should respond with 400
+              And response body contains
+                  | Offset must be omitted or set to a value between 0 and 500. |
+        Examples:
+                  | Value                    |
+                  | -1                       |
+                  | 'zero'                   |
+                  | '5; select * from users' |
+                  | '0)'                     |
+                  | '1%27'                   |
+
+        @ignore
+        Scenario: Ensure clients can't GET information when querying using an offset without providing any limit in the query string
```

Comment:
This scenario actually proves the opposite: that we _can_ get information. Not "can't" 😄

Reply:
Done